packages/app/src/vis-packs/core/visualizations.test.ts

```diff
@@ -72,135 +72,135 @@ describe('Raw', () => {
   const { supportsDataset } = CORE_VIS.Raw;
 
   it('should support any dataset', () => {
-    expect(supportsDataset(scalarInt, mockStore)).toBe(true);
-    expect(supportsDataset(twoDStr, mockStore)).toBe(true);
+    expect(supportsDataset(scalarInt)).toBe(true);
+    expect(supportsDataset(twoDStr)).toBe(true);
```

Comment:
Quick win from using `satisfies` when defining the core and NeXus visualizations.